Package.swift

```diff
@@ -19,6 +19,7 @@ let package = Package(
     .library(name: "HttpPipelineHtmlSupport", targets: ["HttpPipelineHtmlSupport"]),
     .library(name: "HttpPipelineTestSupport", targets: ["HttpPipelineTestSupport"]),
     .library(name: "MediaType", targets: ["MediaType"]),
+    .library(name: "UrlFormEncoding", targets: ["UrlFormEncoding"]),
```

Comment:
I split out the `urlFormEncode` helper into its own package so that we could use it in a few spots...